Small QA issues for 19.10

anthonydillon opened the pull request:
## Done
- Use label pattern for experimental in feature list on /desktop/developer
- Add `u-no-padding--top` to the first p in each download box to align with the Download button on `/download/desktop` and `/download/server`
- Replace the inline styles with class on the /download page as does not look good on small screens
- /download/iot the Ubuntu Core button removed the chevron
- Spliting the Install Ubuntu Server box into its own strip on download/iot/raspberry-pi to add some space

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check done comments look ok

webteam-app commented:
Starting demo at: https://ubuntu-com-canonical-web-and-design-pr-5957.run.demo.haus/

anasereijo commented:
Hi Ant, thank you for all these improvements 👍 🥇 

Just 3 tiny things:

### /desktop/developers

- [ ] remove `--divided` from both lists

- [ ] move the label next to the text. you picked orange as experimental is also something that's in progress or not final right?

![Desktop for developers   Ubuntu](https://user-images.githubusercontent.com/36884067/67003690-6d5d2800-f0d6-11e9-8642-00e381f65ace.png)

### /download

- [ ] can you match the space between Cloud and flavours on mobile or is this a vanilla issue?

<img width="485" alt="Screenshot 2019-10-17 at 12 09 30" src="https://user-images.githubusercontent.com/36884067/67003931-08560200-f0d7-11e9-8167-abf09a8744f0.png">

anthonydillon commented:
@anasereijo thanks, I've addressed all your feedback. Yes, I choice in-progress label as it seemed to closest to experimental: https://docs.vanillaframework.io/patterns/labels/

anasereijo commented:
LGTM @anthonydillon
